splinterd/man/splinterd.1.md

```diff
@@ -269,14 +269,14 @@ OPTIONS
 : Specifies the path and file name for the REST API key.
   (Default: `/etc/splinter/certs/rest_api.key`.)
 
-`--whitelist WHITELIST` `[,...]`
+`--allow_list ALLOW_LIST` `[,...]`
```

Comment:
Should be --allow-list, not --allow_list

Reply:
fixed